server/polar/user/endpoints/license_keys.py

```diff
@@ -0,0 +1,175 @@
+from fastapi import Depends, Query
+from pydantic import UUID4
+
+from polar.benefit.schemas import BenefitID
+from polar.exceptions import NotPermitted, ResourceNotFound, Unauthorized
+from polar.kit.db.postgres import AsyncSession
+from polar.kit.pagination import ListResource, PaginationParamsQuery
+from polar.license_key.schemas import (
+    LicenseKeyActivate,
+    LicenseKeyActivationBase,
+    LicenseKeyActivationRead,
+    LicenseKeyDeactivate,
+    LicenseKeyRead,
+    LicenseKeyValidate,
+    LicenseKeyWithActivations,
+    NotFoundResponse,
+    UnauthorizedResponse,
+    ValidatedLicenseKey,
+)
+from polar.license_key.service import license_key as license_key_service
+from polar.models import LicenseKeyActivation
+from polar.openapi import APITag
+from polar.organization.schemas import OrganizationID
+from polar.postgres import get_db_session
+from polar.routing import APIRouter
+
+from .. import auth
+
+router = APIRouter(prefix="/license-keys", tags=[APITag.documented, APITag.featured])
+
+
+ActivationNotPermitted = {
+    "description": "License key activation not required or permitted (limit reached).",
+    "model": NotPermitted.schema(),
+}
+
+
+@router.get(
+    "/{id}",
+    response_model=LicenseKeyWithActivations,
+    responses={
+        401: UnauthorizedResponse,
+        404: NotFoundResponse,
+    },
+)
+async def get_license_key(
+    auth_subject: auth.UserLicenseKeysRead,
+    id: UUID4,
+    session: AsyncSession = Depends(get_db_session),
+) -> LicenseKeyWithActivations:
+    """Get a license key."""
+    lk = await license_key_service.get_loaded(session, id)
+    if not lk:
+        raise ResourceNotFound()
+
+    user_id = auth_subject.subject.id
+    if user_id != lk.user_id:
+        raise Unauthorized()
+
+    ret = LicenseKeyWithActivations.model_validate(lk)
+    expose_activations = lk.benefit.properties.get("activations", {}).get(
+        "enable_user_admin", False
+    )
+    if not expose_activations:
+        ret.activations = []
+
+    return ret
+
+
+@router.get(
+    "",
+    response_model=ListResource[LicenseKeyRead],
+    responses={
+        401: UnauthorizedResponse,
+        404: NotFoundResponse,
+    },
+)
+async def list_license_keys(
+    auth_subject: auth.UserLicenseKeysRead,
+    pagination: PaginationParamsQuery,
+    organization_id: OrganizationID,
```

Comment:
Same thing as my first comment: with an optional/multi-valued approach, we allow users to get all their license keys across several organizations. But not a breaking change to do it afterwards.